exercises/rectangles/canonical-data.json

```diff
@@ -51,6 +51,33 @@
       "expected": 5
     },
     {
+      "description": "rectangle of height 1 is counted",
+      "input": [
+        "+---+",
+        "+---+"
+      ],
+      "expected": 1
+    },
+    {
+      "description": "rectangle of width 1 is counted",
```

Comment:
Is there a reason why both of these rectangles have their other dimension 3?
Perhaps one of them could be 1x2.

(This may be a frivolous requirement.)